test/mpi/util/mtest_thread.c

```diff
@@ -118,9 +119,20 @@ int MTest_thread_barrier(int nt)
     }
     /* Really need a write barrier here */
     *cntP = *cntP - 1;
+    wait = *cntP > 0;
```

Comment:
How about rename `wait` to `cntP_value` or `counter_value`?

Reply:
In this patch I am using the boolean value of `*cntP > 0` to determine if waiting is necessary. `*_value` is not appropriate IMO, but I am open to other suggestions.

Reply:
Perhaps `bool done` or `bool not_done`...

Reply:
What I see from the code is you are simply extracting the value of `*cntP` into `wait`. Upon reading, the first I need to grasp is "what is it" and then it is "what is it for". Thus I'd like the name to tell what it is first.